src/test/java/lotto/enums/LottoRankTest.java

```diff
@@ -0,0 +1,49 @@
+package lotto.enums;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+class LottoRankTest {
+
+    @DisplayName("로또 랭크를 제대로 판단하는지 확인한다")
+    @Test
+    public void testFirstPrize() {
+        LottoRank rank = LottoRank.determineRank(6, false);
+        assertEquals(LottoRank.FIRST, rank);
+        assertEquals(2_000_000_000L, rank.getPrizeAmount());
+    }
+
+    @DisplayName("로또 랭크를 제대로 판단하는지 확인한다")
```

Comment:
테스트 코드의 이름 (또는 DisplayName)은 최대한 친절하게 자세하게 적어줄 수록 좋다고 생각해요!
물론 저도 아직 잘 안되는 부분입니다...!! ㅎㅎㅎ
가령 이런건 어떨까 해 제안드려 봅니다.

```suggestion
    @DisplayName("5와 true가 입력되면 LottoRank.SECOND가 반환된다.")
```

이와 같은 @DisplayName은 @ParameterizedTest와 결합하여 정말 친절한 이름으로 작성할 수 있더라고요!
한번 학습해 보시면 좋을 것 같습니다!!